addons/sale_project/static/tests/tours/sale_project_tour.js

```diff
@@ -0,0 +1,124 @@
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { stepUtils } from "@web_tour/tour_service/tour_utils";
+import { markup } from "@odoo/owl";
+
+registry.category("web_tour.tours").add("task_create_sol_tour", {
+    test: true,
+    url: "/web",
+    rainbowManMessage: _t("Congratulations! You have successfully created a sales order item on the fly from a Task."),
+    steps: () => [stepUtils.showAppsMenuItem(), {
+    trigger: ".o_app[data-menu-xmlid='project.menu_main_pm']",
+    position: "bottom",
+}, {
+    trigger: ".o-kanban-button-new",
+    extra_trigger: ".o_project_kanban",
+    content: markup(_t("Let\'s create your first <b>project</b>.")),
+    position: "bottom",
+    width: 200,
```

Comment:
that attibute is useful for the content? 

If yes then no need since the purpose of that tour is just for testing the feature works as expected.